tests/test_data_ir.py

```diff
@@ -0,0 +1,62 @@
+import numpy as np
+import nir
+
+
+def test_generate_time_gridded_data():
+    spikes = np.random.randint(0, 2, size=(10, 10, 10)).astype(bool)
+    dt = 0.1
+    gridded = nir.TimeGriddedData(spikes, dt)
+    node = nir.NIRNodeData({"spikes": gridded})
+    graph = nir.NIRGraphData({"node": node})  # noqa: F841
```

Comment:
It's great to test that you can create the objects, but it would be good to also verify the content. A simple test could be to ensure the information is preserved. It's basic, but something like `np.allclose(graph.nodes["node"].data, spikes)` would make sure we won't have unexpected regressions.